node/lib/cmd/commit.js

```diff
@@ -118,7 +130,8 @@ exports.executeableSubcommand = co.wrap(function *(args) {
     // this time.
 
     if (repoStatus.areUncommittableSubmodules()) {
-        process.stderr.write(Status.printRepoStatus(repoStatus, relCwd));
```

Comment:
`console.error` is shorter but appends a newline that would be redundant in this case.